src/components/common/YourCompany/EntityName.vue

```diff
@@ -228,15 +227,15 @@
       <v-col cols="7">
         <div class="name-request-applicant-info">
           <span class="subtitle">Name: </span>
-          <span class="info-text">{{ nrApplicant.fullName }}</span>
```

Comment:
The NR object no longer has fullName or fullAddress so we now use a computed property to format it.